samples/interactive-tutorials/src/main/java/product/AddFulfillmentPlaces.java

```diff
@@ -56,6 +56,15 @@ public static void addFulfillmentPlaces(String productName, String placeId)
             .setAllowMissing(true)
             .build();
 
+    // To send an out-of-order request assign the invalid AddTime here:
+    // Instant instant = LocalDateTime.now().minusDays(1).toInstant(ZoneOffset.UTC);
+    // Timestamp previousDay = Timestamp.newBuilder()
```

Comment:
We strongly prefer not having commented out code, as it's not tested or compiled.